README.md

```diff
@@ -186,6 +186,7 @@ If you would you like to start your own language implementation of IPFS, check o
 | Ruby | https://github.com/Fryie/ipfs-ruby | |
 | Swift | https://github.com/NeoTeo/ipfs-osx-service | |
 | PHP | https://github.com/cloutier/php-ipfs-api | |
+| | https://github.com/digitalkaoz/php-ipfs | |
```

Comment:
Could you rename your repo to `php-ipfs-api`? We use (or try to) `ipfs-api` for client libraries and just `ipfs` for actual implementations. Otherwise, I'm afraid that users will be confused.